python/pyproject.toml

```diff
@@ -112,8 +107,15 @@ skip = [
     "__init__.py",
 ]
 
-[tool.setuptools]
-license-files = ["LICENSE"]
+[tool.scikit-build]
```

Comment:
Can we sort the keys in this section?

Reply:
I have no way to gauge "importance". Alphabetical is fine with me.